### DISCUSSION TITLE
[GH Community/Projects&Issues] UI placeholder issue

### DISCUSSION THREAD

**Discussion by xmarkgs** [community/community#121483] [Discussions]
### Select Topic Area

Bug

### Body

Bug location: https://github.com/orgs/community/discussions/new?category=projects-and-issues

`<p class="CommentBox-placeholder">` tag representing the placeholder element is overflowing its container when `<textarea>` on the same level is active.

Image attached.
![Screenshot 2024-04-28 at 17 28 23](https://github.com/community/community/assets/45517995/77b5200a-d753-44d0-9e67-b29c5748511c)

**Comment by LiteBrite82** [comment]
This community is managed by GitHub employees, but it is not a support forum and there is no SLA or guarantee each post will get a response from a community manager or product leader.

If your discussion provided product feedback or a feature request we encourage you to review the "Making Suggestions" [guidelines](https://github.com/community/community#making-suggestions) in the repo readme including: "Once you kick off a discussion, the GitHub product team will evaluate the feedback but will not be able to respond to every submission."

Thank you for contributing and continuing to support other users in the GitHub Community Discussions.
